ethical-advertising-works.rst

```diff
@@ -0,0 +1,84 @@
+Ethical Advertising Works
+=========================
+
+It has been two years since we first launched :doc:`ads on Read the Docs <ads-on-read-the-docs>`.
+We figured it was time to report on the results that we've seen,
+and say thanks to those who have helped us along the way.
+
+The Results
+-----------
+
+To put it simply:
+
+    **Ethical Advertising works**.
+
+In Q1 2018,
+our estimated revenue from advertising is $75,000.
+This is enough to fund our small team of 4 people.
+We are working at below market rates for San Francisco,
+but we are happy to be able to sustain Read the Docs with this model.
+
+When we started building our ethical advertising model,
+we didn't know how it would work out.
+We didn't know much about the ad industry,
+and we were doing things different than everyone else.
+
+We are happy to report that you can build advertising that is a win/win/win for all parties:
+
+* Read the Docs wins because we generate revenue, allowing us to sustain our operations
+* Advertisers win because they are able to reach our audience of developers, helping to grow their business.
+* Users win because their privacy is maintained, and they continue to have access to our service.
+
+
+Next Steps
+----------
+
+We are proud to have pioneered this successful user-centric advertising model.
+
+We hope that we will act as an example for other projects that are building advertising into their business model.
+Developers have the power to shift the ad industry to be more user and privacy focused.
+We have seen other advertising platforms talk about how their ads are ethical,
```

Comment:
I don't know specifically if others are calling out their ads as ethical but they certainly are raising advertiser standards and being slightly more transparent about what they track. Shifting this conversation at all in favor of respecting users is a big win!